subworkflows/nf-core/fasta_gxf_busco_plot/main.nf

```diff
@@ -0,0 +1,172 @@
+                                                    Channel.fromList(val_lineages)
+                                                )
+                                                | map { meta, fasta, mode, lineage ->
+                                                    [
+                                                        meta + [ mode:mode, lineage:lineage ],
+                                                        fasta, mode, lineage
+                                                    ]
+                                                }
+                                                | combine(
+                                                    ch_db_path
+                                                )
+                                                | combine(
+                                                    ch_config_path
+                                                )
+
+    BUSCO_ASSEMBLY(
+        ch_busco_assembly_inputs.map { meta, fasta, mode, lineage, db, config -> [ meta, fasta ] },
+        ch_busco_assembly_inputs.map { meta, fasta, mode, lineage, db, config -> mode },
+        ch_busco_assembly_inputs.map { meta, fasta, mode, lineage, db, config -> lineage },
+        ch_busco_assembly_inputs.map { meta, fasta, mode, lineage, db, config -> db ?: [] },
+        ch_busco_assembly_inputs.map { meta, fasta, mode, lineage, db, config -> config ?: [] }
+    )
+
+    ch_assembly_batch_summary                   = BUSCO_ASSEMBLY.out.batch_summary
+    ch_assembly_short_summaries_txt             = BUSCO_ASSEMBLY.out.short_summaries_txt
+    ch_assembly_short_summaries_json            = BUSCO_ASSEMBLY.out.short_summaries_json
+    ch_assembly_full_table                      = BUSCO_ASSEMBLY.out.full_table
+    ch_versions                                 = ch_versions.mix(BUSCO_ASSEMBLY.out.versions.first())
+
+    // MODULE: BUSCO_GENERATEPLOT as PLOT_ASSEMBLY
+    ch_assembly_plot_summary                    = ch_assembly_short_summaries_txt
+                                                | map { meta, txt ->
+                                                    def lineage_name = meta.lineage.split('_odb')[0]
+                                                    [
+                                                        "short_summary.specific.${meta.lineage}.${meta.id}_${lineage_name}.txt",
+                                                        txt.text
+                                                    ]
+                                                }
+                                                | collectFile
+
+    PLOT_ASSEMBLY( ch_assembly_plot_summary.collect() )
+
+    ch_assembly_png                             = PLOT_ASSEMBLY.out.png
+    ch_versions                                 = ch_versions.mix(PLOT_ASSEMBLY.out.versions)
+
+    // MODULE: GFFREAD as EXTRACT_PROTEINS
+    ch_gffread_inputs                           = ! ( val_mode == 'geno' || val_mode == 'genome' )
```

Comment:
```suggestion
    ch_gffread_inputs                           = ! ( val_mode in [ 'geno', 'genome' ] )
```
!in should also work, but I think the language server complains ( or it did the last time I used it ).

Reply:
Switched to `val_mode !in [ 'geno', 'genome' ]` and the server did not complain.